src/loginServices.js

```diff
@@ -514,6 +514,9 @@ export async function logout(okapiUrl, store) {
       localStorage.removeItem(SESSION_NAME);
       localStorage.removeItem(RTR_TIMEOUT_EVENT);
 
+      // Clear saved entry path so that subsequent logins will use default base URL.
+      removeUnauthorizedPathFromSession();
+
```

Comment:
This doesn't look correct. We preserve the last-active URL for three reasons: 
1. when active sessions end due to [IST](https://github.com/folio-org/stripes-core/blob/3aef1a056c2932b94e2ed179e9eb9fc8c6922bef/src/components/SessionEventContainer/SessionEventContainer.js#L34)
2. when active sessions end due to [FLST](https://github.com/folio-org/stripes-core/blob/3aef1a056c2932b94e2ed179e9eb9fc8c6922bef/src/components/SessionEventContainer/SessionEventContainer.js#L47)
3. when inactive sessions are started from a bookmark in AuthnLogin, as you noticed here

This change will break 1. and 2. 

Instead of clearing storage on the way out during logout, I think we need to clear it on the way in, right after it's read in [OIDCRedirect](https://github.com/folio-org/stripes-core/blob/3aef1a056c2932b94e2ed179e9eb9fc8c6922bef/src/components/OIDCRedirect.js).

Reply:
Makes sense, I'll make the correction. Thanks for pointing out the other use cases.